src/lib/wpack.js

```diff
@@ -75,7 +75,25 @@ const run = configs =>
     });
   });
 
+/**
+ * Runs webpack with an array of configurations in series
+ * @param {array} configs Array of webpack configurations
+ * @returns {Promise} Webpack stats
+ */
+const runSeries = configs =>
```

Comment:
As stated before, you have to call `run` in the reduce to have it run in series, no?

Something like: 
```
const runSeries = R.reduce((promise, config) => promise.then(() => run(config)), Promise.resolve());
```

Reply:
I'm running `run` on line 88 (https://github.com/goldwasserexchange/serverless-plugin-webpack/pull/44/files/3f3101f9d0509d4b9eefe159bac4ad93f9414332#diff-4084f9978b63349375192b14cbac5bcfR88).

Reply:
Yes, but the problem with promises is that they are launched as soon as they are created. So when you call run in the `map` your are running the configuration in parallel (with a very small start delay). You can solve this by reducing on the config array.

Reply:
@jogold I'm not returning a `Promise` in the `map`, I'm returning a function which creates a `Promise`. So it actually does run in series :).

Reply:
You are right! My mistake, sorry. Still, don't you prefer the more concise version that uses `ramda` (like the rest of the code) : `const runSeries = R.reduce((promise, config) => promise.then(() => run(config)), Promise.resolve());`